config/src/test/java/org/hyperledger/besu/config/GenesisConfigFileTest.java

```diff
@@ -59,7 +59,7 @@ void shouldLoadMainnetConfigFile() {
 
   @Test
   void shouldLoadDevelopmentConfigFile() {
-    final GenesisConfigFile config = GenesisConfigFile.development();
+    final GenesisConfigFile config = GenesisConfigFile.fromResource("/dev.json");
```

Comment:
this is less discoverable and brittle - wrapping the resource location with a reusable and easier-to-discover method name is cleaner IMO

Reply:
I removed the `GenesisConfigFile.development()` method since it is only meant for testing, and so does not fit well being part of production code.